jmix-flowui/flowui/src/main/java/io/jmix/flowui/impl/DialogsImpl.java

```diff
@@ -613,8 +612,25 @@ public String getMaxHeight() {
         }
 
         @Override
-        public void open() {
+        public Dialog build() {
+            DialogAction okAction = new DialogAction(DialogAction.Type.OK);
```

Comment:
Message Dialog has only OK button which doesn't depend on any setting of the builder.  With current implementation, sequential calls of the `build()` method produce button duplicates:

```
@Subscribe(id = "testBtn", subject = "clickListener")
public void onTestBtnClick(final ClickEvent<JmixButton> event) {
    Dialogs.MessageDialogBuilder builder = dialogs.createMessageDialog()
            .withText("Test");
    Dialog dialog = builder.build();
    dialog = builder.open();
}
```

<img width="438" alt="Screenshot 2024-09-02 at 11 49 37" src="https://github.com/user-attachments/assets/964ca9df-13fe-4e5b-806b-5a4cab97c9a5">

For example, `OptionsDialogBuilder` removes all buttons in the `build()` and inits them again. But in case of Message Dialog, as mentioned above, the OK button is persistent.